torch_xla/csrc/runtime/env_hash.cc

```diff
@@ -0,0 +1,105 @@
+#include "torch_xla/csrc/runtime/env_hash.h"
+
+#include <iostream>
+#include <sstream>
+
+#include "torch_xla/csrc/runtime/sys_util.h"
+
+namespace torch_xla {
+namespace runtime {
+namespace hash {
+
+namespace {
+static const std::string XLA_FLAG_PREFIX = "--xla";
+
+// Taken from JAX:
+// https://github.com/google/jax/blob/8ee5811/jax/_src/cache_key.py#L325-L346
+static const std::set<std::string> FlagsToExclude = {
```

Comment:
not that it really matters, but is there a reason to use `set` instead of `unordered_set`?

Reply:
No reason, I'll switch to `unordered_set`